src/hal/src/sensors_cf2.c

```diff
@@ -348,9 +351,13 @@ static void sensorsDeviceInit(void)
   // To low DLPF bandwidth might cause instability and decrease agility
   // but it works well for handling vibrations and unbalanced propellers
   // Set output rate (1): 1000 / (1 + 1) = 500Hz
-  mpu6500SetRate(1);
+  mpu6500SetRate(0);
```

Comment:
This will change setting to 1kHz sampling output right? Please update comment above if that is the case. Also does 1kHz sample output improve anything, latency maybe?